### DISCUSSION TITLE
Base branch is getting deleted after job completion

### DISCUSSION THREAD

**Discussion by jpraxsh** [mend/renovate-ce-ee#616] [Q&A]
This is my config file (config.js). My base branch is getting deleted after Renovate runs successfully and the MR is created.

module.exports = {
    extends: ['config:recommended'],
    baseBranches: ['dev_bot_jp'],
    onboarding: false,
    requireConfig: false,
    dependencyDashboard: true,
    automerge: false,
    packageRules: [
      {
        matchFileNames: ['^package.json$'],
        matchUpdateTypes: ['patch', 'minor'],
        automerge: false
      }
    ],
    npm: { 
      fileMatch: ['^package.json$']
    }
  };

**Comment by rarkins** [comment]
It's impossible to diagnose without logs

**Reply by jpraxsh** [reply]
ok Thanks.
